qiskit/providers/ibmq/runtime/__init__.py

```diff
@@ -102,11 +102,12 @@
     qc.measure_all()
 
     # Execute the circuit using the "circuit-runner" program.
-    program_inputs = {'circuits': circuit, 'measurement_error_mitigation': True}
+    param = provider.runtime.program(program_id="circuit-runner").parameter
+    param.param1 = 123
```

Comment:
Looking at this example I feel having `.parameter` as a property is a bit awkward. When you retrieve the attribute of an object multiple times without changing the state of the object, you typically expect to get the same result each time. But here if you do 
```
program = provider.runtime.program(program_id="circuit-runner")
program.parameter.param1 = 123
program.parameter.param2 = 456
```
each `.parameter` actually returns a new `ParameterNamespace` instance. And passing `program.parameter` to `run()` will be missing `param1`. 

I do like that each call returns a new instance - makes it easier to call the same program with different parameters. So perhaps just make `parameters()` a method?

Reply:
Fixed!